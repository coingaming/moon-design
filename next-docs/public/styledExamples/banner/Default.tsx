import { Banner } from '@heathmont/moon-core';

const Example = () => (
  <Banner title="Title here" description="Lorem ipsum dolor..." />
);

export default Example;
