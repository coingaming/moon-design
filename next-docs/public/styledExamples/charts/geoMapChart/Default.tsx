// @ts-nocheck
import React from 'react';
import { Filter, GeoMapChart } from '@heathmont/moon-charts';
import { SingleItemSelect } from '@heathmont/moon-components';

const Example = () => {
  const data = [
    { label: 'Estonia', code: 'EST', value: 120 },
    { label: 'Russia', code: 'RUS', value: 100 },
    { label: 'Belarus', code: 'BLR', value: 80 },
    { label: 'Argentina', code: 'ARG', value: 60 },
    { label: 'USA', code: 'USA', value: 40 },
  ];

  const filter = (
    <Filter title="by GGR">
      {() => (
        <SingleItemSelect
          onChange={() => {}}
          value="ggr"
          options={[
            { label: 'Active players', value: 'actives' },
            { label: 'GGR', value: 'ggr' },
            { label: 'Bets', value: 'bets' },
          ]}
        />
      )}
    </Filter>
  );

  return (
    <GeoMapChart
      title="Top Countries"
      data={data}
      filter={filter}
      hasUpdates={true}
      onUpdate={() => {}}
      onShare={() => {}}
      onExpand={() => {}}
    />
  );
};

export default Example;
