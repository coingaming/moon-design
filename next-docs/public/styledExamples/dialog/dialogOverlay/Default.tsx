import React from 'react';
import { DialogOverlay } from '@heathmont/moon-components';
import { Button } from '@heathmont/moon-core';

const Example = () => {
  const [showDialog, setShowDialog] = React.useState(false);

  return (
    <>
      <Button variant="tertiary" onClick={() => setShowDialog(true)}>
        Open Dialog
      </Button>
      <DialogOverlay
        isOpen={showDialog}
        onDismiss={() => setShowDialog(false)}
      />
    </>
  );
};

export default Example;
