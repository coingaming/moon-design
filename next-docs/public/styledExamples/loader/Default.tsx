import React from 'react';
import { Loader } from '@heathmont/moon-core';

const Example = () => <Loader />;

export default Example;
