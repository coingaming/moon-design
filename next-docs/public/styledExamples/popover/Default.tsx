import React, { useCallback, useEffect, useState } from 'react';
import { Popover } from '@heathmont/moon-popover';
import { Footer, Heading, Text, Button } from '@heathmont/moon-core';

const Example = () => {
  const [open, setOpen] = useState(false);

  const HandlerOpen = useCallback(
    (e: any) => {
      e.preventDefault();
      setOpen(false);
    },
    [open]
  );

  return (
    <Popover
      padding={0}
      placement="bottom"
      isOpen={open}
      onVisibilityChange={(status) => {
        setOpen(status);
        console.log('status: ', status);
      }}
      content={
        <div>
          <div className="p-8 space-y-4">
            <Heading size={20}>Agents</Heading>
            <Text size={14}>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
              enim ad minim veniam, quis nostrud exercitation ullamco laboris
              nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
              reprehenderit in voluptate velit esse cillum dolore eu fugiat
              nulla pariatur.
            </Text>
          </div>

          <Footer
            primButton={
              <Button fullWidth size="small" onClick={HandlerOpen}>
                OK
              </Button>
            }
            secButton={
              <Button
                fullWidth
                variant="secondary"
                size="small"
                onClick={HandlerOpen}
              >
                Cansel
              </Button>
            }
            tertButton={
              <Button
                fullWidth
                variant="tertiary"
                size="small"
                onClick={HandlerOpen}
              >
                Decline
              </Button>
            }
            isDivider
            size="small"
          />
        </div>
      }
    >
      <Button>Toggle Popover</Button>
    </Popover>
  );
};

export default Example;
