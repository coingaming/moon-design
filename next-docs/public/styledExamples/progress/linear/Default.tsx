import React from 'react';
import { ProgressLinear } from '@heathmont/moon-components';

const Example = () => (
  <>
    <ProgressLinear
      value={30}
      text={<span>Progress</span>}
      backgroundColor="hit.100"
    />
    <ProgressLinear
      value={30}
      text={<span>Progress</span>}
      backgroundColor="hit.100"
      height={5}
    />
  </>
);

export default Example;
