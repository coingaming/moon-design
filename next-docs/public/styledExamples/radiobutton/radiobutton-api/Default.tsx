import React from 'react';
import {
  RadioButtonLabel,
  RadioButtonInput,
  ListItem,
} from '@heathmont/moon-core';

const Example = () => (
  <div className="flex flex-col">
    <h4 className="mb-2">Please select your age:</h4>
    <div className="mb-2">
      <RadioButtonLabel id="age1">
        <ListItem
          backgroundColor="goku.100"
          elementRight={
            <RadioButtonInput name="age" ariaLabel="0 - 30" id="age1" />
          }
        >
          0 - 30
        </ListItem>
      </RadioButtonLabel>
    </div>
    <div className="mb-2">
      <RadioButtonLabel id="age2">
        <ListItem
          backgroundColor="goku.100"
          elementRight={
            <RadioButtonInput name="age" ariaLabel="0 - 30" id="age2" />
          }
        >
          31 - 60
        </ListItem>
      </RadioButtonLabel>
    </div>
    <div className="mb-2">
      <RadioButtonLabel id="age3">
        <ListItem
          backgroundColor="goku.100"
          elementRight={
            <RadioButtonInput name="age" ariaLabel="61 - 100" id="age3" />
          }
        >
          31 - 60
        </ListItem>
      </RadioButtonLabel>
    </div>
  </div>
);

export default Example;
