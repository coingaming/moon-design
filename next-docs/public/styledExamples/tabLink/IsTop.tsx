import { TabLink } from '@heathmont/moon-core';

const Example = () => <TabLink isTop={true} />;

export default Example;
