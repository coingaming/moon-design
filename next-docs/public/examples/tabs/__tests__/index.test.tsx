/**
 * @jest-environment jsdom
 */

import React from 'react';
import { moonDesignLight, ThemeProvider } from '@heathmont/moon-themes';
import renderer from 'react-test-renderer';

import Default from '../Default';
import Fill from '../Fill';
import FillContainer from '../FillContainer';
import IsTop from '../IsTop';
import Small from '../Small';
import Variant from '../Variant';

const renderWithTheme = (component: JSX.Element) => (
  <ThemeProvider theme={moonDesignLight}>{component}</ThemeProvider>
);

describe('Tabs', () => {
  it('renders', () => {
    const testRenderer = renderer.create(renderWithTheme(<Default />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Fill', () => {
    const testRenderer = renderer.create(renderWithTheme(<Fill />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Fill Container', () => {
    const testRenderer = renderer.create(renderWithTheme(<FillContainer />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Is Top', () => {
    const testRenderer = renderer.create(renderWithTheme(<IsTop />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Small', () => {
    const testRenderer = renderer.create(renderWithTheme(<Small />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Variant', () => {
    const testRenderer = renderer.create(renderWithTheme(<Variant />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
