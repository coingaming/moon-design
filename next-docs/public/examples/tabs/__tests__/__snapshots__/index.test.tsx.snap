// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs renders 1`] = `
<nav
  className="TabNav-sc-1uwascp-0 jGhhiE"
>
  <a
    className="SkipLink-sc-1rku5na-0 hlIhqd"
    href="#nav-test-id-1"
  >
    Skip to content
  </a>
  <ul
    className="Reel-sc-1q0v9zl-0-ul TabList-b0t4ul-0 GYukh"
  >
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#2"
      >
        Featured
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#3"
      >
        Today
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#4"
      >
        Tomorrow
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#5"
      >
        Future
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#6"
      >
        Outright
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#7"
      >
        Leagues
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#8"
      >
        News
      </a>
    </li>
  </ul>
  <span
    id="nav-test-id-1"
  />
</nav>
`;

exports[`Tabs renders Fill 1`] = `
<nav
  className="TabNav-sc-1uwascp-0 jGhhiE"
>
  <a
    className="SkipLink-sc-1rku5na-0 hlIhqd"
    href="#nav-test-id-1"
  >
    Skip to content
  </a>
  <ul
    className="Reel-sc-1q0v9zl-0-ul TabList-b0t4ul-0 GYukh"
  >
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#2"
      >
        Featured
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#3"
      >
        Today
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#4"
      >
        Tomorrow
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#5"
      >
        Future
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#6"
      >
        Outright
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#7"
      >
        Leagues
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#8"
      >
        News
      </a>
    </li>
  </ul>
  <span
    id="nav-test-id-1"
  />
</nav>
`;

exports[`Tabs renders Fill Container 1`] = `
<nav
  className="TabNav-sc-1uwascp-0 gyCWLE"
>
  <a
    className="SkipLink-sc-1rku5na-0 hlIhqd"
    href="#nav-test-id-1"
  >
    Skip to content
  </a>
  <ul
    className="Reel-sc-1q0v9zl-0-ul TabList-b0t4ul-0 GYukh eYA-DLK"
  >
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#2"
      >
        Featured
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#3"
      >
        Today
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#4"
      >
        Tomorrow
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#5"
      >
        Future
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#6"
      >
        Outright
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#7"
      >
        Leagues
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-vls6mb-0 dDYhJq"
        href="#8"
      >
        News
      </a>
    </li>
  </ul>
  <span
    id="nav-test-id-1"
  />
</nav>
`;

exports[`Tabs renders Is Top 1`] = `
<nav
  className="TabNav-sc-1uwascp-0 jGhhiE"
>
  <a
    className="SkipLink-sc-1rku5na-0 hlIhqd"
    href="#nav-test-id-1"
  >
    Skip to content
  </a>
  <ul
    className="Reel-sc-1q0v9zl-0-ul TabList-b0t4ul-0 GYukh"
  >
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 ldfUgR"
        href="#2"
      >
        Featured
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 ldfUgR"
        href="#3"
      >
        Today
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 ldfUgR"
        href="#4"
      >
        Tomorrow
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 ldfUgR"
        href="#5"
      >
        Future
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 ldfUgR"
        href="#6"
      >
        Outright
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 ldfUgR"
        href="#7"
      >
        Leagues
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 ldfUgR"
        href="#8"
      >
        News
      </a>
    </li>
  </ul>
  <span
    id="nav-test-id-1"
  />
</nav>
`;

exports[`Tabs renders Small 1`] = `
<nav
  className="TabNav-sc-1uwascp-0 jGhhiE"
>
  <a
    className="SkipLink-sc-1rku5na-0 hlIhqd"
    href="#nav-test-id-1"
  >
    Skip to content
  </a>
  <ul
    className="Reel-sc-1q0v9zl-0-ul TabList-b0t4ul-0 GYukh"
  >
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 cIUSWF"
        href="#2"
        size="small"
      >
        Featured
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 cIUSWF"
        href="#3"
        size="small"
      >
        Today
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 cIUSWF"
        href="#4"
        size="small"
      >
        Tomorrow
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 cIUSWF"
        href="#5"
        size="small"
      >
        Future
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 cIUSWF"
        href="#6"
        size="small"
      >
        Outright
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 cIUSWF"
        href="#7"
        size="small"
      >
        Leagues
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 cIUSWF"
        href="#8"
        size="small"
      >
        News
      </a>
    </li>
  </ul>
  <span
    id="nav-test-id-1"
  />
</nav>
`;

exports[`Tabs renders Variant 1`] = `
<nav
  className="TabNav-sc-1uwascp-0 jGhhiE"
>
  <a
    className="SkipLink-sc-1rku5na-0 hlIhqd"
    href="#nav-test-id-1"
  >
    Skip to content
  </a>
  <ul
    className="Reel-sc-1q0v9zl-0-ul TabList-b0t4ul-0 GYukh"
  >
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#2"
      >
        Featured
        <span
          className="TabLink__CountWrapper-u6qu8y-1 kVKWpR"
        >
          00
        </span>
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#3"
      >
        Today
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#4"
      >
        <div
          className="TabLink__ElementLeftWrapper-u6qu8y-0 bbgJEx"
        >
          <svg
            className="sc-avest"
            fill="none"
            height="1em"
            viewBox="0 0 32 32"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clipRule="evenodd"
              d="M29.007 17.303l-3.392-6.085c-.348-.448-.783-.806-1.218-.895-.348-.09-.696 0-1.044.268-.348.269-.522.627-.609 1.074l3.132 5.728c2.174 4.027 1.478 7.875-2 11.455-.262.268-.436.447-.697.626a6.272 6.272 0 003.914-2.058c3.567-3.67 3.393-7.607 1.914-10.113zm-4.175.716l-3.131-5.638c-.261-.358-.61-.626-1.044-.716-.261-.09-.522 0-.783.18-.522.447-.696 1.073-.261 1.879l1.566 3.49a.71.71 0 01-.174.716c-.174.179-.522.09-.696-.09l-8.177-8.502c-.434-.447-1.217-.447-1.652 0-.261.269-.348.537-.348.895s.087.627.348.806l4.87 5.011c.262.18.262.537.088.716a.452.452 0 01-.696.09s-.087 0-.087-.09l-4.871-4.922-1.392-1.432c-.522-.537-1.217-.537-1.652 0-.435.448-.435 1.253 0 1.7l1.391 1.433 4.871 5.011c.174.18.174.537 0 .806-.174.268-.522.179-.783 0L7.35 14.35c-.435-.447-1.218-.447-1.653 0-.261.269-.348.537-.348.806 0 .357.087.626.348.805l2.087 2.148 3.045 3.132c.174.268.174.537-.087.716-.174.179-.435.179-.696 0l-3.044-3.132c-.261-.269-.522-.358-.87-.358-.61 0-1.131.537-1.131 1.163 0 .358.087.627.348.895l5.915 6.086c3.218 3.31 8.089 5.011 11.742 1.253 3.045-3.133 3.654-6.355 1.827-9.845zM13.524 8.891c.087-.268.174-.537.435-.805a1.554 1.554 0 012.088 0l2.783 2.863a2.767 2.767 0 00-.696 2.685l-4.61-4.743zm1.305-6.444l-1.653.806 2.088 2.506-.435-3.312zm7.306.806l-1.652-.806-.348 3.312 2-2.506zm-4.523 1.7L16.742 2h1.74l-.87 2.953z"
              fill="currentColor"
              fillRule="evenodd"
            />
          </svg>
        </div>
        Tomorrow
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#5"
      >
        <div
          className="TabLink__ElementLeftWrapper-u6qu8y-0 bbgJEx"
        >
          <svg
            className="sc-avest"
            fill="none"
            height="1em"
            viewBox="0 0 32 32"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clipRule="evenodd"
              d="M29.007 17.303l-3.392-6.085c-.348-.448-.783-.806-1.218-.895-.348-.09-.696 0-1.044.268-.348.269-.522.627-.609 1.074l3.132 5.728c2.174 4.027 1.478 7.875-2 11.455-.262.268-.436.447-.697.626a6.272 6.272 0 003.914-2.058c3.567-3.67 3.393-7.607 1.914-10.113zm-4.175.716l-3.131-5.638c-.261-.358-.61-.626-1.044-.716-.261-.09-.522 0-.783.18-.522.447-.696 1.073-.261 1.879l1.566 3.49a.71.71 0 01-.174.716c-.174.179-.522.09-.696-.09l-8.177-8.502c-.434-.447-1.217-.447-1.652 0-.261.269-.348.537-.348.895s.087.627.348.806l4.87 5.011c.262.18.262.537.088.716a.452.452 0 01-.696.09s-.087 0-.087-.09l-4.871-4.922-1.392-1.432c-.522-.537-1.217-.537-1.652 0-.435.448-.435 1.253 0 1.7l1.391 1.433 4.871 5.011c.174.18.174.537 0 .806-.174.268-.522.179-.783 0L7.35 14.35c-.435-.447-1.218-.447-1.653 0-.261.269-.348.537-.348.806 0 .357.087.626.348.805l2.087 2.148 3.045 3.132c.174.268.174.537-.087.716-.174.179-.435.179-.696 0l-3.044-3.132c-.261-.269-.522-.358-.87-.358-.61 0-1.131.537-1.131 1.163 0 .358.087.627.348.895l5.915 6.086c3.218 3.31 8.089 5.011 11.742 1.253 3.045-3.133 3.654-6.355 1.827-9.845zM13.524 8.891c.087-.268.174-.537.435-.805a1.554 1.554 0 012.088 0l2.783 2.863a2.767 2.767 0 00-.696 2.685l-4.61-4.743zm1.305-6.444l-1.653.806 2.088 2.506-.435-3.312zm7.306.806l-1.652-.806-.348 3.312 2-2.506zm-4.523 1.7L16.742 2h1.74l-.87 2.953z"
              fill="currentColor"
              fillRule="evenodd"
            />
          </svg>
        </div>
        Future
        <span
          className="TabLink__CountWrapper-u6qu8y-1 kVKWpR"
        >
          00
        </span>
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#6"
      >
        Outright
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#7"
      >
        Leagues
      </a>
    </li>
    <li
      className="Tabs__Tab-fs06ov-0"
    >
      <a
        className="Container-j1xqi2-0 iUlyKi"
        href="#8"
      >
        News
      </a>
    </li>
  </ul>
  <span
    id="nav-test-id-1"
  />
</nav>
`;
